Backend/routes/routes.py

```diff
@@ -1,4 +1,69 @@
 from fastapi import APIRouter
+
+
+from Backend.model.model import User
+from Backend.config.database import collection, db
+from Backend.schemas.schema import user_addition_serializer
+from typing import Union
+from bson import ObjectId
+from fastapi import HTTPException
+
+user_addition_router = APIRouter()
```

Comment:
can we change this to user_router instead of user_addition_router since we will be using it for all CRUD operations

Reply:
changed to user_router